### PR TITLE
Simon add 1

### PR DESCRIPTION
**Pull request by SimonZHao0429**
Simon add 1Simon add 1Simon add 1

### CODE REVIEW

File: src/core.c [hunk 1]
```diff
@@ -1,3 +1,4 @@
 ZhaoChengBo start it
 Simon is a volunteer,thanks!
 ZhaoChengBo add 1
+Simon add 1
```
Comment (ZhaoChengBo) [inline src/core.c]: Do you think it's right??

Comment (SimonZHao0429) [reply src/core.c]: Yes !!! Fuck

**Comment by ZhaoChengBo** [comment]
Thanks

**Comment by ZhaoChengBo** [comment]
Thanks@SimonZhao0429

**Pull request merged**